codegen/bin/coins

```diff
@@ -59,7 +59,9 @@ erbs = [
   {'template' => 'CoinInfoData.cpp.erb', 'folder' => 'src/Generated', 'file' => 'CoinInfoData.cpp'},
   {'template' => 'registry.md.erb', 'folder' => 'docs', 'file' => 'registry.md'},
   {'template' => 'hrp.cpp.erb', 'folder' => 'src/Generated', 'file' => 'TWHRP.cpp'},
-  {'template' => 'hrp.h.erb', 'folder' => 'include/TrustWalletCore', 'file' => 'TWHRP.h'}
+  {'template' => 'hrp.h.erb', 'folder' => 'include/TrustWalletCore', 'file' => 'TWHRP.h'},
+  {'template' => 'TWChainId.h.erb', 'folder' => 'include/TrustWalletCore', 'file' => 'TWChainId.h'},
```

Comment:
We generate `TWEthereumChainId` for backward compatibility, no need to add ChainId enum because it's not existed before and not recommended to use.